<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CentralWidget</class>
 <widget class="QWidget" name="CentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>17 Helper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Error</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retryBasicDownloadButton">
       <property name="text">
        <string>Retry Download Sets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retryTemplateButton">
       <property name="text">
        <string>Retry Download Template</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="MTGAHelperGroup">
     <property name="title">
      <string>MTGAHelper</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="pwdLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="pwdEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="remembePwdCheck">
        <property name="text">
         <string>Remember Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="savePwdWarningLabel">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>170</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>170</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">Your password will be stored in plain text in %1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="setsGroup">
     <property name="title">
      <string>Data Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="formatsCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Sets</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="draftableSetsCheck">
            <property name="text">
             <string>Show only draftable sets</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="allSetsButton">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="noSetButton">
            <property name="text">
             <string>Select None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="ratingTimeGroup">
          <property name="title">
           <string>Ratings Date Limits</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>From</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="ratingFromEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>83</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>To</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="ratingToEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="toTodayCheck">
             <property name="text">
              <string>Today</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="setsView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Rating Based On</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ratingBasedCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ratingBasedButton">
            <property name="toolTip">
             <string>Definitions</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/help.svg</normaloff>:/icons/help.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Notes Contain</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="onlyRatiosCheck">
            <property name="text">
             <string>Show Only Ratios</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="allMetricsButton">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="noMetricsButton">
            <property name="text">
             <string>Select None</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="notesView"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customRatingsGroup">
     <property name="title">
      <string>Ratings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Search Card</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchCardEdit"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>252</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="MTGAHTab">
         <attribute name="title">
          <string>MTGA Helper Ratings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableView" name="ratingsView">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="SLTab">
         <attribute name="title">
          <string>17 Lands Ratings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableView" name="slRatingsView">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Custom ratings</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTableView" name="customRatingsView">
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="progressLabel">
       <property name="text">
        <string notr="true">Action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelUploadButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download Ratings from 17 Lands</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearRatingsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear Ratings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Upload Ratings to MTGA Helper</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
